handlers/search.go

```diff
@@ -69,16 +69,21 @@ func SearchAhead(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	searchType := vars["type"]
 	search := vars["search"]
-	if !utils.IsValidEnsDomain(search) {
-		search = strings.Replace(search, "0x", "", -1)
-	}
 	var err error
 	logger := logger.WithField("searchType", searchType)
 	var result interface{}
 
+	strippedSearch := strings.Replace(search, "0x", "", -1)
+	lowerStrippedSearch := strings.ToLower(strippedSearch)
+
+	if len(strippedSearch) == 0 {
+		_ = json.NewEncoder(w).Encode(result)
+		return
+	}
```

Comment:
This change causes the search to find nothing if only "0x" is entered. As the results that were found were not really meaningful, I believe we can accept that though.